.github/workflows/ci.yml

```diff
@@ -1,78 +1,18 @@
 name: CI
+
 on:
   push:
     branches:
-      - main
+      - master
```

Comment:
We don't have a master branch in this repo :) Lets revert back to `main`

Reply:
Addressed in [4ae71d8](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/205/commits/4ae71d81c47f3fb1ef327b86f3c4ff4cf43c4e83)